packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -152,6 +152,7 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
       calloutWidth,
       calloutMaxWidth,
       finalHeight,
+      hideOverflow,
```

Comment:
this could be `hideOverflow: !!finalHeight` so it falls back if you don't provide the prop.